### PR TITLE
Add new utils scripts, including one to center a group in the box

### LINKED ISSUE #187: Center group of atoms in the box
I'm not sure if this is a feature we would like to have in the main MD driver or if it's something better in `utils`, but I think we should provide a way of centering a group of atoms in the center of the box.

I've been performing simulations with micelles and often the micelle move in the box and end up broken across the pbcs. So having a way of translating stuff to have a whole micelle in the center of the box would be nice.

### PR DESCRIPTION
**Pull request by hmcezar**
This PR closes #187.

I added 3 new scripts to utils:

- `center_group.py` works by passing a group of beads, e.g. `1-1234` or `1 3 6-100`, calculates the centroid of this group of atoms for each frame, and, finally, centers this group in the box. The final trajectory has the group in the center of the box in each frame of the trajectory.
- `nm2a_h5md.py` converts coordinates from nm to angstrom for a `.H5` output trajectory.
- `pdb2hdf5.py` receives a `.pdb` and based on `.txt` files with charges, connectivity etc, generates a `.hdf5` input.

### CODE REVIEW

**Comment by hmcezar** [comment]
If someone could review / test the scripts (especially the one that centers the groups) it would be great!

File: utils/center_group.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# based on https://stackoverflow.com/a/6512463/3254658
+def parse_bead_list(string):
+    m = re.match(r'(\d+)(?:-(\d+))?$', string)
+    if not m:
+        raise argparse.ArgumentTypeError("'" + string + "' is not a range of number. Expected forms like '0-5' or '2'.")
+    start = m.group(1)
+    end = m.group(2) or start
+    if int(end) < int(start):
+        raise argparse.ArgumentTypeError(f"Start value ({start}) should be larger than final value ({end})")
+    return list(range(int(start,10), int(end,10)+1))
```
Comment (Lun4m) [inline utils/center_group.py:16]:
```suggestion
    start = int(m.group(1), base=10)
    end = int(m.group(2), base=10) or start
    if end < start:
        raise argparse.ArgumentTypeError(f"Start value ({start}) should be larger than final value ({end})")
    return list(range(start, end + 1))
```

File: utils/center_group.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    f_in.close()
+    f_out.close()
+
+
+if __name__ == '__main__':
+    description = 'Center geometric center of beads in the box for each frame in a .H5 trajectory'
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('h5md_file', type=str, help='input .H5MD file name')
+    parser.add_argument('--beads', type=parse_bead_list, nargs='+', required=True,
```
Comment (Lun4m) [inline utils/center_group.py:71]:
```suggestion
    parser.add_argument('-b', '--beads', type=parse_bead_list, nargs='+', required=True,
```

File: utils/center_group.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                        help='bead list to center (e.g.: 1-100 102-150)')
+    parser.add_argument('--out', type=str, default=None, dest='out_path',
```
Comment (Lun4m) [inline utils/center_group.py:73]:
```suggestion
    parser.add_argument('-o', '--out', type=str, default=None, dest='out_path',
```

**Comment by Lun4m** [comment]
Very minor suggestions for `center_group.py`.

**Pull request merged**